### PR TITLE
Log new process

### PR DESCRIPTION
**Pull request by chajain**
Log every new process started/failed on the endpoint (SENS-12)

### CODE REVIEW

File: artifacts/definitions/Linux/Events/ProcessStatuses.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Linux.Events.ProcessStatuses
+description: |
+  This artifact collects New process status logs from the Linux kernel.
```
Comment (guangyee) [inline artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: How about this?

This artifact collects new processes created by non-root users from the Linux kernel audit events.

File: artifacts/definitions/Linux/Events/ProcessStatuses.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
```
Comment (guangyee) [inline artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: Do we need to give a zypper example as well?

**Comment by guangyee** [comment]
Looks good! I am able to test this PR in my local vagrant development environment. A couple of observations:

1. if there's an existing static rule (in /etc/audit/rules.d/) which does the same thing but with a different tag, all the audit events will be using that tag instead.
2. I ran into an this error while testing the PR, `vql: receive failed: no buffer space available`, which seem to be (according to Nikolay) related to netlink queue buffer out of space. This resulted in the VM became unresponsive. However, I haven't been able to reproduce this error since the VM reboot.

File: artifacts/definitions/Linux/Events/ProcessStatuses.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+sources:
+  - query: |
+     // Install the auditd rule if possible.
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-d", "task,never"])
+
+     LET _ <= SELECT * FROM execve(argv=[pathToAuditctl, "-a",
+          "exit,always", "-F", "arch=b64", "-F", "auid!=-1", "-F",
+          "euid!=0", "-S", "execve", "-k", "proexec"])
```
Comment (guangyee) [inline artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: Suggest prefix the tag with `vrr_` to distinguish the ones added by velociraptor. i.e. `vrr_proexec`.

Comment (Firstyear) [reply artifacts/definitions/Linux/Events/ProcessStatuses.yaml]: Agreed :)

**Pull request closed without merging**